static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-liveblog.scss

```diff
@@ -93,9 +93,49 @@
             }
         }
     }
+}
 
+//pillar colour overrides
+@mixin overide-live-blog-colours($pillar, $color1, $color2) {
+    .fc-container:not(.fc-container--story-package) .fc-item--type-liveblog.fc-item--pillar-#{$pillar} {
+        background-color: $color1;
+        .fc-item__kicker {
+            background-color: $color1;
+            color: $color2;
+            border: 0;
+            &::before {
+                background-color: $color2;
+            }
+            &::after {
+                background-color: $color2;
+            }
+        }
+        .fc-item__container::before {
+            background-color: $color2;
+        }
+        .live-pulse-icon::before {
+            background-color: $color2;
+        }
+        .u-faux-block-link--hover {
+            background-color: darken($color1, 5%);
+            .fc-item__kicker,
+            .fc-item__container::before {
```

Comment:
this will result in a selector like `.a:not(.b) .c.d .e .f::before` which is pretty full on – can this be reduced?